llvm/test/tools/llvm-objcopy/ELF/Inputs/srec-elf-sections-err.yaml

```diff
@@ -0,0 +1,22 @@
+!ELF
+FileHeader:
+  Class:           ELFCLASS64
+  Data:            ELFDATA2LSB
+  Type:            ET_EXEC
+  Machine:         EM_X86_64
+Sections:
+  - Name:            .text1
+# Part of section data is in 32-bit address range
+# and part isn't.
+    Type:            SHT_PROGBITS
+    Flags:           [ SHF_ALLOC, SHF_EXECINSTR ]
```

Comment:
`SHF_EXECINSTR` is unnecessary for the test, I believe, so please delete it (same below).

Keep your test data to a minimum for what your test actually needs.